### DISCUSSION TITLE
Handle onClick callback on Progress component segments

### DISCUSSION THREAD

**Discussion by Azzerty23** [mantinedev/mantine#2580] [Feature requests]
Hi amazing Mantine community, I would like to filter table data by clicking on a Progress segment.
Is it possible to add an onClick handler to Progress segment ?

EDIT 20/10/2022 : has been added in [Version: 5.6.0](https://github.com/mantinedev/mantine/commit/fe744cabd3eec5ab46411e0922dffa54a33111d4#diff-d091a540167aa69cd64eb09e4cc1d1657ec9f4c94c06bc25fceadbd594771b84) ✅

**Comment by tybirk-accuranker** [comment]
+1 for this and for the same thing on RingProgress. 
I am using Progress component for a simple bar chart and RingProgress for a donut chart. In both cases, click events would be very nice to have for the individual sections :-)
